skxray/fitting/xrf_model.py

```diff
@@ -1117,12 +1116,12 @@ def linear_spectrum_fitting(spectrum, params, elemental_lines=None,
         if np.sum(total_y[:, i]) == 0:
             continue
         result_dict.update({total_list[i]: total_y[:, i]})
-        area = 0
+        area = None
         if area_option:
             area = out[i]*element_area[total_list[i]]
```

Comment:
If I were going to optimize this function, this is not where I would start.

Also, don't start optimizing until you know a) that it matters b) where the bottle neck is.